src/Widgets/DataLabelWidget/DataLabelWidgetEditingConfig.ts

```diff
@@ -13,6 +13,7 @@ provideEditingConfig(DataLabelWidget, {
       values: [
         { value: 'text', title: 'Text' },
         { value: 'currency', title: 'Currency' },
+        { value: 'datetime', title: 'Date time' },
```

Comment:
* [ ] I don't think "date time" is a common term. How about "Date/time" or just "Date"?